src/chapters/09/main.tex

```diff
@@ -382,47 +358,89 @@ \section{Solving with Python}\label{sec:solving-with-python}
 print(solution_with_swap_stops)
 \end{pyin}
 
-giving:
+gives:
 
 \begin{pyout}
 [0, 7, 2, 8, 5, 3, 1, 9, 12, 11, 4, 10, 6, 0]
 \end{pyout}
 
+This has a cost:
+
 \begin{pyin}
-solution_with_reverse_path = run_neighbourhood_search(
+cost = get_cost(
+    tour=solution_with_swap_stops,
     distance_matrix=distance_matrix,
-    number_of_stops=number_of_stops,
-    iterations=number_of_iterations,
-    seed=seed,
-    neighbourhood_operator=reverse_path,
 )
-print(solution_with_reverse_path)
+print(cost)
 \end{pyin}
 
-giving:
+\begin{pyout}
+362
+\end{pyout}
+
+Therefore, using this particular algorithm, a pretty good route it found, with a
+total distance of 362.
+
+It is important to note that this may not be the optimal route, and tweaking the
+algorithm slightly may produce better solutions.
+For example, one way to do this is to use a different neighbourhood operator.
+One example is instead of swapping two stop, revering the path between those two
+stops. The \mintinline{python}{reverse_path} function does this:
```

Comment:
```suggestion
It is important to note that this may not be the optimal route, and different algorithms
may produce better solutions.
For example, one way to modify the algorithm is to use a different neighbourhood operator.
Instead of swapping two stops, reverse the path between those two
stops. The \mintinline{python}{reverse_path} function does this:
```